### ISSUE TITLE
Hey again! Filters to !random command?

### ISSUE THREAD

**Issue by Zackman634** [irbyk/Plex-Discord-Bot_V2#10]
I was wondering if it's possible to add a way to narrow down the selection of songs it picks from when it selects randomly. Also, on that note, could you add a !gerne command like you have a !mood command?

**Comment by irbyk** [comment]
Can you give me some details of what you have in mind ?

**Comment by Zackman634** [comment]
I was thinking a command that was something along the lines of !random <modifier to narrow song selection>
Such as; !random happy, which would pull from a random selection of happy songs
!random rock, which would pull from a selection of rock music

**Comment by irbyk** [comment]
Doesn't the `!mood` command do the trick ?

**Comment by Zackman634** [comment]
Yes. I apologize, I got myself confused. But, the second command would still be valid

**Comment by irbyk** [comment]
Don’t apologize for that ^^. I just add some command (`!library` for example to select which plex library you want for the bot to use when you use `!random` or `!mood`). 
But if you want a another way to random your song you can create a playlist with the bot and play it randomly `!playlist play <name of your playlist> r`. Note it’s a bot specific playlist, not the plex one. If you want to use a plex playlist you can type `!playlist plex <name of the playlist>` and then `!shuffle` to shuffle the queue.
